src/appengine.h

```diff
@@ -0,0 +1,31 @@
+#ifndef AKTUALIZR_LITE_APP_ENGINE_H_
+#define AKTUALIZR_LITE_APP_ENGINE_H_
+
+#include <string>
+
+class AppEngine {
+ public:
+  struct App {
+    std::string name;
+    std::string uri;
+  };
+
+ public:
+  virtual bool fetch(const App& app) = 0;
+  virtual bool install(const App& app) = 0;
+  virtual bool run(const App& app) = 0;
+  virtual void remove(const App& app) = 0;
+  virtual bool isInstalled(const App& app) = 0;
```

Comment:
This doesn't seem to be used *yet*. I say drop it until you introduce code that will use it.

Reply:
It's used in composeappengine.cc/h, The methods are defined as 'override' there. If I remove them the code won't compile.

Reply:
but its dead code there also, right? i see it *defined* and has a TODO, but its not actually referenced by any code yet?

Reply:
no, it's not, except isInstalled().